public class ExceptionMain {
	public static void main(String[] args) {
		
		int j = 20;
		int i = 0;
		
		try {
		    System.out.println(" =>" + (1/0));
		} catch (Exception e) {
			
			System.out.println("Division par zero impossible (" + e.getMessage() + ").");			
		    e.printStackTrace();
		}
		finally{
		    System.out.println("Action faite systématiquement");
		}
	}
}
